template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Startup.fs

```diff
@@ -8,33 +8,21 @@ open Microsoft.AspNetCore.Builder
 open Microsoft.AspNetCore.Hosting
 open Microsoft.Extensions.Configuration
 open Microsoft.Extensions.DependencyInjection
-open Microsoft.Extensions.Logging
 
 
 type Startup private () =
 
-    new (env: IHostingEnvironment) as this =
+    new (configuration: IConfiguration) as this =
         Startup() then
-
-        let builder =
-            ConfigurationBuilder()
-                .SetBasePath(env.ContentRootPath)
-                .AddJsonFile("appsettings.json", optional = false, reloadOnChange = true)
-                .AddJsonFile((sprintf "appsettings.%s.json" (env.EnvironmentName)), optional = true)
-                .AddEnvironmentVariables()
-
-        this.Configuration <- builder.Build()
+        this.Configuration <- configuration
```

Comment:
This is also a style issue, but if you use the primary constructor syntax _and_ the Configuration property is _not_ a convention that the framework relies on (which I doubt since Startup does not implement an Interface), then you do not need the Configuration property since in F#, primary constructor arguments are in scope for all the members of a type (so you could just use the configuration argument anywhere you would have used this.Configuration).